srcpkgs/python3-networkx/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-networkx'
 pkgname=python3-networkx
-version=2.6.2
+version=2.6.3
 revision=1
 wrksrc="networkx-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools unzip"
-depends="python3-decorator"
```

Comment:
Now this package doesn't depend on python3 transitively, so we need to add an explicit dependency.

Reply:
Ah, right.  I've added python3 as the depends.